build-js.sh

```diff
@@ -587,26 +587,29 @@ echo "building binaryen.js"
 
 "$EMSCRIPTEN/em++" \
   $EMCC_ARGS \
-  $BINARYEN_SRC/binaryen-c.cpp \
+  "$BINARYEN_SRC/binaryen-c.cpp" \
   shared.bc \
-  -I$BINARYEN_SRC/ \
+  "-I$BINARYEN_SRC/" \
   -s EXPORTED_FUNCTIONS=[${EXPORTED_FUNCTIONS}] \
-  -o $BINARYEN_BIN/binaryen${OUT_FILE_SUFFIX}.js \
+  -o "$BINARYEN_BIN/binaryen${OUT_FILE_SUFFIX}.js" \
   -s MODULARIZE_INSTANCE=1 \
   -s 'EXPORT_NAME="Binaryen"' \
-  --post-js $BINARYEN_SRC/js/binaryen.js-post.js
+  --post-js "$BINARYEN_SRC/js/binaryen.js-post.js"
+# note: does not need --pre-js currently
 
 echo "building binaryen-wasm.js"
 
 "$EMSCRIPTEN/em++" \
   $EMCC_ARGS \
-  $BINARYEN_SRC/binaryen-c.cpp \
+  "$BINARYEN_SRC/binaryen-c.cpp" \
   shared.bc \
-  -I$BINARYEN_SRC/ \
+  "-I$BINARYEN_SRC/" \
   -s EXPORTED_FUNCTIONS=[${EXPORTED_FUNCTIONS}] \
-  -o $BINARYEN_BIN/binaryen-wasm${OUT_FILE_SUFFIX}.js \
+  -o "$BINARYEN_BIN/binaryen-wasm${OUT_FILE_SUFFIX}.js" \
   -s BINARYEN=1 \
   -s 'BINARYEN_METHOD="native-wasm"' \
   -s MODULARIZE_INSTANCE=1 \
   -s 'EXPORT_NAME="Binaryen"' \
-  --post-js $BINARYEN_SRC/js/binaryen.js-post.js
+  -s NO_EXIT_RUNTIME=0 \
```

Comment:
Without, there is a warning when running binaryen-wasm.js. Is that correct?

Reply:
Something about `atexits`? That's expected when assertions are on, but not in an optimized build. So we shouldn't see it here, I don't think. Or was it something else?

Reply:
Let me double check

Reply:
You are right, this must have been caused by something else.